pages/index.tsx

```diff
@@ -17,6 +17,7 @@ import {
   getVariableDebtBalances,
 } from '../utils/balances'
 import { ethers } from 'ethers'
+import Step_progess from './stepprogress/step_progress'
```

Comment:
We should be consistent in naming style, camel case, pascal case or snake case

Suggest sticking with pascal case for components